src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
+
+(function () {
+
+  let BACKUP = {};
+
+  QUnit.config.testTimeout = 6400;
+
+  // runs before any tests begin
+  QUnit.begin(() => {
+    // disable storage persistence
+    chrome.storage.local.set = () => {};
```

Comment:
Originally I was concerned that this instance of `chrome.storage.local.set` was shared by the tests, and the real privacy badger, so overriding this would effect the real PB. But this isn't the case (!!).